clientd3d/d3drender_skybox.h

```diff
@@ -14,9 +14,27 @@
 #ifndef _D3DRENDERSKYBOX_H
 #define _D3DRENDERSKYBOX_H
 
-void D3DRenderSkyBoxBegin();
-void D3DRenderSkyBox(room_type* room, Draw3DParams* params, room_contents_node* pRNode, 
-	int& angleHeading, int& anglePitch);
+struct SkyboxRenderParams {
+    LPDIRECT3DVERTEXDECLARATION9 vertexDeclaration;
+    d3d_driver_profile driverProfile;
+    mutable d3d_render_pool_new renderPool;
```

Comment:
Ideally these would not be mutable.  const can be kind of like a virus, but can you do a quick check to see if the parameters to D3DRenderPoolReset, D3DRenderSkyboxDraw, D3DCacheFill, D3DCacheFlush can be const without further changes?  If so, you should be able to remove the mutable tag.